app/cdap/components/GridTable/components/Breadcrumb/styles.ts

```diff
@@ -25,4 +26,18 @@ export const useStyles = makeStyles({
     marginRight: '30px',
     marginLeft: '34px',
   },
+  weight400: {
```

Comment:
can u change the classname to more meaningful, it is not just about weight = 400
plz update

Reply:
Done